interfaces/PA-App/src/app/personal-components/phone-number/phone-number.component.ts

```diff
@@ -14,22 +16,35 @@ export class PhoneNumberComponent extends PersonalComponent {
   public choiceMade = false;
   public phoneNumber: number;
   public phone: any;
+  public ngo: string;
+  public did: string;
 
   constructor(
     private conversationService: ConversationService,
     public translate: TranslateService,
+    public paData: PaDataService,
+    public programService: ProgramsServiceApiService,
   ) {
     super();
   }
 
-  ngOnInit() { }
+  ngOnInit() {
+    this.paData.retrieve('ngo').then(value => {
```

Comment:
Here this phone-number should be related/connected to a program, somehow.

Or we could 'simplify' it all by setting the 'author/actor` of this component to be the 121-`system`.